apps/condo/domains/news/components/NewsForm/SelectSharingAppControl.tsx

```diff
@@ -0,0 +1,148 @@
+                        image: {
+                            src: CONDO_APP_PREVIEW_ICON_URL,
+                            style: CARD_ICON_STYLE,
+                        },
+                    }}
+                />
+            </div>
+
+            { sharingAppContexts.map( ctx => {
+                // Replace all ' ' with non-breaking space (\u00A0)
+                const sharingAppName = get(ctx, ['app', 'newsSharingConfig', 'name'], '').replaceAll(' ', '\u00A0')
+
+                const sharingAppIcon = get(ctx, ['app', 'newsSharingConfig', 'icon', 'publicUrl'], SHARING_APP_FALLBACK_ICON)
+                const sharingAppPreviewIcon = get(ctx, ['app', 'newsSharingConfig', 'previewPicture', 'publicUrl'], SHARING_APP_FALLBACK_PREVIEW_ICON)
+
+                return (
+                    <div key={ctx.id} style={cardStyle}>
+                        <CardCheckbox
+                            header={{
+                                headingTitle: sharingAppName,
+                                image: {
+                                    size: 'small',
+                                    src: sharingAppIcon,
+                                },
+                            }}
+                            body={{
+                                description: intl.formatMessage({ id: AppDescriptionLabelId }, { appName: sharingAppName }),
+                                image: {
+                                    src: sharingAppPreviewIcon,
+                                    style: CARD_ICON_STYLE,
+                                },
+                            }}
+                            checked={selectedSharingApps.has(ctx.id)}
+                            onChange={(checked) => handleSelectSharingApp({ value: ctx.id, checked })}
+                        />
+                    </div>
+                )
+            })}
+
+            { sharingAppContexts.length === 0 && (
+                <div style={cardStyle}>
+                    <Card>
+                        <Row gutter={[0, 10]}>
+                            <Typography.Title level={3}>{OtherAppsLabel}</Typography.Title>
+                            <Typography.Paragraph>{OtherAppsDescriptionLabel}</Typography.Paragraph>
+                            <div
+                                style={{
+                                    alignItems: 'center',
+                                    alignSelf: 'stretch',
+                                    display: 'flex',
+                                    justifyContent: 'center',
+                                    width: '100%',
+                                    backgroundColor: colors.gray[1],
+                                    borderRadius: '12px',
+                                }}
+                            >
+                                <img
+                                    src={PROMO_APP_PREVIEW_ICON}
+                                    style={{
```

Comment:
move styles to const

Reply:
Used CARD_ICON_STYLE const here, left style above untouched, since it is used only once